src/globus_sdk/_testing/data/flows/create_flow.py

```diff
@@ -0,0 +1,32 @@
+from responses import matchers
+
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+from ._common import TWO_HOP_TRANSFER_FLOW_DOC
+
+_two_hop_transfer_create_request = {
+    k: TWO_HOP_TRANSFER_FLOW_DOC[k]
+    for k in [
+        "title",
+        "definition",
+        "input_schema",
+        "subtitle",
+        "description",
+        "flow_viewers",
+        "flow_starters",
+        "flow_administrators",
+        "keywords",
+    ]
+}
+RESPONSES = ResponseSet(
+    metadata={
+        "params": _two_hop_transfer_create_request,
+    },
+    default=RegisteredResponse(
+        service="flows",
+        path="/flows",
+        method="POST",
+        json=TWO_HOP_TRANSFER_FLOW_DOC,
+        match=[matchers.json_params_matcher(params=_two_hop_transfer_create_request)],
```

Comment:
Hmm, I'm not sure where I stand.

My gut tells me that having `request_json_match` as a single input field on all the examples we have ther will make it look to new developers like `RegisteredResponse` takes a single matching thing when in reality you can register as many matchers as you'd like for the same response.


On the other hand, we're not actually taking away any of that multi-matcher functionality and it is a slightly cleaner interface.

I'd probably lean very slightly against for the first reason, just because the slight obfuscation seems a little more costly than the value of the simplified structure. But it's such a slight leaning that I would be happy either way.

Reply:
> I'd probably lean very slightly against for the first reason, just because the slight obfuscation seems a little more costly than the value of the simplified structure.

Let's go with that then, no change for now.
I'll mark this resolved.

👍 for introducing the `match=...` usage in the first place!